Frontend/pages/index.js

```diff
@@ -16,7 +16,7 @@ export default function Home() {
       <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
         <Grid container spacing={3}>
           {/* Chart */}
-          <Grid item xs={12} md={8} lg={9}>
```

Comment:
for the homepage, I would just leave the order table and remove everything else

Reply:
Done with commit "remove deposit in homepage"